pages/checkin.tsx

```diff
@@ -24,6 +24,14 @@ const CheckingPage: React.FC<{}> = () => {
   })
   const [doCheckout] = useMutation(api.checkoutTicket)
 
+  const [area, setArea] = React.useState<api.AreaResponse>()
+  React.useEffect(() => {
+    areaId &&
+      api.fetchArea(areaId).then((area) => {
+        setArea(area)
+      })
```

Comment:
I would prefer a more descriptive syntax

```suggestion
    if (!areaId) return
    api.fetchArea(areaId).then((area) => {
      setArea(area)
    })
```